<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="signButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Вход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>361</width>
      <height>29</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSigned">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти из аккаунта</string>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>signButton</zorder>
   <zorder>regButton</zorder>
   <zorder>labelSigned</zorder>
   <zorder>exitButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
